### DISCUSSION TITLE
The output activation function of SwinUnetr?

### DISCUSSION THREAD

**Discussion by justinzhang528** [Project-MONAI/tutorials#1400] [Q&A] [answered]
Dear monai community,

I would like to know which activation function is the output of [SwinUnetr](https://docs.monai.io/en/stable/_modules/monai/networks/nets/swin_unetr.html#SwinUNETR) used for? In addition i found that SimpleInferer() returns tensor with values range from -61 to 60. I'm a little confused why the output values are like this because sigmod and softmax output values are between 0 and 1.
![image](https://github.com/Project-MONAI/tutorials/assets/47877537/dabbfbbd-bf81-45c0-923a-669258723f84)
(my model output is 2 channels)

Thanks in advance!

**Comment by KumoLiu** [comment]
Hi @justinzhang528, we didn't add an activation function in the network, you can add any activation function by yourself using `Activations` transforms.
Hope it can help you, thanks!

**Reply by justinzhang528** [reply, marked as answer]
I got it, thanks!
